build.yaml

```diff
@@ -19,16 +19,16 @@ builders:
     auto_apply: root_package
     build_to: cache
     runs_before:
-      - build_test|test_bootstrap
-      - test_html_builder|aggregator
-      - test_html_builder|dart_test_yaml
-      - test_html_builder|templates
+      - build_test:test_bootstrap
```

Comment:
I'm not familiar with this delimiter change. Was this introduced in some specific build* package that we should be careful about?

Reply:
No either format works, but the colon is apparently recommended. The JSON schema expects it so I just did this to silence the red underlines :)